homeassistant/components/light/device_action.py

```diff
@@ -93,3 +101,33 @@
             )
 
     return actions
+
+
+async def async_get_action_capabilities(hass: HomeAssistant, config: dict) -> dict:
+    """List action capabilities."""
+    if config[CONF_TYPE] != toggle_entity.CONF_TURN_ON:
+        return {}
+
+    registry = await entity_registry.async_get_registry(hass)
+    entry = registry.async_get(config[ATTR_ENTITY_ID])
+    state = hass.states.get(config[ATTR_ENTITY_ID])
+
+    supported_features = 0
+
+    if state:
+        supported_features = state.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
+    elif entry:
+        supported_features = entry.supported_features
+
+    if not supported_features & SUPPORT_BRIGHTNESS:
+        return {}
+
+    return {
+        "extra_fields": vol.Schema(
+            {
+                vol.Optional("brightness", default=100): vol.All(
```

Comment:
Drop default

Reply:
Hmm, but without a default we get a UI like this:
![image](https://user-images.githubusercontent.com/14281572/75382376-ab672580-58da-11ea-80c9-905e97c03c1e.png)

This is quite strange, the action is "turn on", but with zero brightness?

Reply:
This comment makes perfect sense:
> If it's not there, we shouldn't pass it to the service. That way a light will turn on in the previous brightness.

But not sure how to represent that in the UI?

Reply:
According to discussion on discord, an optional parameter without default value could be shown like this:
![image](https://user-images.githubusercontent.com/14281572/75476548-137d4080-599b-11ea-9e9e-16401b9cc2c6.png)

Reply:
Frontend updated to support the checkbox: home-assistant/home-assistant-polymer#5003